pkgs/servers/monitoring/prometheus/kthxbye.nix

```
@@ -0,0 +1,53 @@
+{ pkgs
+, lib
+, buildGo119Module
+, fetchFromGitHub
+, gnumake
+, go_1_19
+, glibc
+, coreutils
+, nixosTests
+}:
+
+buildGo119Module rec {
+  pname = "kthxbye";
+  version = "0.15";
+
+  src = fetchFromGitHub rec {
+    owner = "prymitive";
+    repo = "kthxbye";
+    rev = "v${version}";
+    hash = "sha256-N1MzutjzLk9MnE1b7dKRsiS7LL4Nb61+NpmjTBPGohI=";
+  };
+
+  vendorHash = "sha256-PtINxblqX/wxJyN42mS+hmwMy0lCd6FcQgmBnxTUdcc=";
+
+  nativeBuildInputs = [
+    gnumake
+    go_1_19
+    glibc
+    coreutils
+  ];
+
+  buildPhase = ''
+    make
+  '';
```

```suggestion
  buildPhase = ''
    make -j$NIX_BUILD_CORES
  '';
```
Also I am pretty sure that we do not need to use the makefile and I would highly prefer to not use it and use our tooling
```suggestion
```

---

Build fails if I use the default builder, I think, because they're building different go modules in different directories, and are running custom build params. 

Makefile for reference: https://github.com/prymitive/kthxbye/blob/main/Makefile